canonical-web-and-design/ubuntu.com
# Update copy for certifications page

## Done

- Updated copy for certifications

## QA

- Go to the [demo](https://ubuntu-com-11537.demos.haus/credentialing) and verify the page against the [copy doc](https://docs.google.com/document/d/1GrdkS9pQHKjb92ecKQepXV78XoVw3PKCmueTOGrIdbo)

## Issue / Card

Fixes https://github.com/canonical-web-and-design/commercial-squad/issues/558
